lib/version.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
-class AutoHCK
-  VERSION = '0.6.2'
+# AutoHCK module
+module AutoHCK
+  class AutoHCK
+    VERSION = '0.6.2'
```

Comment:
I think such changes should the propagate version.

Reply:
Added commit with the version bump